File: Seção 13 - Linguagem Python/Exercícios/Exercicio_6.py
N = int(input("Quantos numeros voce vai digitar? "))

vet: [float] = [0 for x in range(N)]
soma = 0

for i in range(0, N):
    vet[i] = float(input("Digite um numero: "))

print()

print("VALORES = ", end="")
for i in range(0, N):
    print(f"{vet[i]:.1f}  ", end="")
print()

for i in range(0, N):
    soma = vet[i] + soma
print(f"SOMA = {soma:.2f}")

media = soma / N
print(f"MEDIA = {media:.2f}")

File: Seção 11 - Linguagem Java/Exercícios/Exercicio_6.java
import java.util.Scanner;

public class Exercicio_6 {

	public static void main(String[] args) {
		
		Scanner sc = new Scanner(System.in);
		
		System.out.print("Quantos numeros voce vai digitar? ");
		int n = sc.nextInt();
		
		double[] vet = new double[n];
		
		for (int i = 0; i < n; i++) {
			System.out.print("Digite um numero: ");
			vet[i] = sc.nextDouble();
		}
		
		
		System.out.println();
		
		System.out.print("VALORES = ");
		
		for (int i = 0; i < n; i++) {
			System.out.print(vet[i] + " ");
		}
		
		System.out.println();
		
		double soma = 0;
		
		for (int i = 0; i < n; i++) {
			soma = soma + vet[i];
		}
		
		System.out.println("SOMA = " + String.format("%.2f", soma));
		
		double media = soma / n;
		
		System.out.println("MEDIA = " + String.format("%.2f", media));
		
		
		sc.close();
		
	}

}
